src/app/navigation/Home.screen.tsx

```diff
@@ -8,30 +15,35 @@ export type HomeRouteParams = undefined;
 
 export const Home = () => {
   const [movies, setMovies] = useState([]);
+  const [nbMovies, setNbMovies] = useState<number>();
 
   useEffect(() => {
-    const fetchMovies = async () => {
-      try {
-        const response = await fetch(API_URL);
-        const json = await response.json();
-        setMovies(json.results);
-      } catch (error) {
-        console.error(error);
-      }
-    };
-
-    fetchMovies();
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((json) => setMovies(json.results));
   }, []);
 
+  useEffect(() => {
```

Comment:
This useEffect will generate too much refresh:
the fetch will generate a refresh when you call setMovies.
Then this new useEffect will be called and it will call setNbMovies which will also refresh the state.

you can just remove the state nbMovies state and add: 
const nbMovies = movies.length
in the body of the functionnal component